### DISCUSSION TITLE
${__from} and ${__to} parameters in url link do not refresh

### DISCUSSION THREAD

**Discussion by pawtar90** [grafana/grafana#44577] [General]
Hello, I use grafana 8.3.4. I have url link with ${__from} and ${__to} parameters but they do not refresh in the link according to set refresh rate. Grafana refresh button also not work. Parameters refresh only when I use browser reshresh (F5). Could you help me with this issue?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
